RIRImageResizing/Classes/NSObject/RIRResizeImageOperationParameters.m

```diff
@@ -0,0 +1,44 @@
+//
+//  RIRResizeImageOperationParameters.m
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import "RIRResizeImageOperationParameters.h"
+
+#import <ResplendentUtilities/RUConditionalReturn.h>
+
+
+
+
+
+@implementation RIRResizeImageOperationParameters
+
+#pragma mark - NSObject
+-(instancetype)init
+{
+    kRUConditionalReturn_ReturnValueNil(YES, YES);
+    return [self init_with_newSize:CGSizeZero
+                        resizeMode:UIImage_RIRResizing_ResizeMode_AspectFit];
+}
+
+#pragma mark - init
+-(instancetype)init_with_newSize:(CGSize)newSize
+                      resizeMode:(UIImage_RIRResizing_ResizeMode)resizeMode
+{
+    kRUConditionalReturn_ReturnValueNil(newSize.height < 0 || newSize.width < 0, NO);
```

Comment:
This should be less than or equal to zero, since no one should be requesting a new image with a height/width of 0.

Also, assertion should be YES, we want to catch that b.s.